libs/milvus/langchain_milvus/vectorstores/milvus.py

```diff
@@ -856,6 +853,77 @@ def add_texts(
             logger.debug("Nothing to insert, skipping.")
             return []
 
+        return self.add_embeddings(
+            texts=texts,
+            embeddings=embeddings,
+            metadatas=metadatas,
+            timeout=timeout,
+            batch_size=batch_size,
+            ids=ids,
+            **kwargs,
+        )
+
+    def add_embeddings(
+        self,
+        texts: List[str],
+        embeddings: List[List[Union[float, List[float]]]],
+        metadatas: Optional[List[dict]] = None,
+        timeout: Optional[float] = None,
+        batch_size: int = 1000,
+        *,
+        ids: Optional[List[str]] = None,
+        **kwargs: Any,
+    ) -> List[str]:
+        """Insert text data with embeddings vectors into Milvus.
```

Comment:
docstr seems exactly the same as the one of add_texts

Reply:
Updated